### ISSUE TITLE
Enter key in editor not working with MacOSX and Android Studio 4.0 Canary

### ISSUE THREAD

**Issue by Bombo** [ant-druha/intellij-powershell#39]
This applies to an editor window (like Kotlin, XML, etc), not just PS scripts. Everything is working fine, until I enable this plugin.

Same for regular enter key and the numpad one.

**Comment by ant-druha** [comment]
Does disabling the plugin help? Could you attach idea.log (Help | Show Log in ... action)? Thanks.

**Comment by Bombo** [comment]
Yes, disabling the plugin solves it. I'll provide you with logs shortly.

**Comment by Bombo** [comment]
I'll attach the full log, but this seems to be the part that shows the error:
```log
PowerShellStringLiteralExpression has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 [Plugin: com.intellij.plugin.adernov.powershell] 
com.intellij.diagnostic.PluginException: While loading class com.intellij.plugin.powershell.psi.PowerShellStringLiteralExpression: com/intellij/plugin/powershell/psi/PowerShellStringLiteralExpression has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 [Plugin: com.intellij.plugin.adernov.powershell]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:221)
	at com.intellij.ide.plugins.cl.PluginClassLoader.tryLoadingClass(PluginClassLoader.java:165)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:73)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at com.intellij.plugin.powershell.ide.injection.PowerShellLanguageInjector.elementsToInjectIn(PowerShellLanguageInjector.kt:26)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.getInjectorMap(InjectedLanguageManagerImpl.java:169)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.processInjectableElements(InjectedLanguageManagerImpl.java:144)
	at com.intellij.codeInsight.daemon.impl.InjectedGeneralHighlightingPass.getInjectedPsiFiles(InjectedGeneralHighlightingPass.java:176)
	at com.intellij.codeInsight.daemon.impl.InjectedGeneralHighlightingPass.collectInformationWithProgress(InjectedGeneralHighlightingPass.java:92)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:429)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1109)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:421)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:397)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:204)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:395)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: java.lang.UnsupportedClassVersionError: com/intellij/plugin/powershell/psi/PowerShellStringLiteralExpression has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:642)
	at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:335)
	at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:330)
	at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:294)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:218)
	... 26 more
2019-11-21 11:28:10,035 [  36061]  ERROR - aemon.impl.PassExecutorService - Android Studio 4.0 Canary 4  Build #AI-192.6817.14.36.6008643 
2019-11-21 11:28:10,035 [  36061]  ERROR - aemon.impl.PassExecutorService - JDK: 1.8.0_212-release; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o 
2019-11-21 11:28:10,035 [  36061]  ERROR - aemon.impl.PassExecutorService - OS: Mac OS X 
2019-11-21 11:28:10,035 [  36061]  ERROR - aemon.impl.PassExecutorService - Plugin to blame: PowerShell version: 2.0.3 
2019-11-21 11:28:10,035 [  36061]  ERROR - aemon.impl.PassExecutorService - Last Action:  

```

**Comment by Bombo** [comment]
[idea.log](https://github.com/ant-druha/intellij-powershell/files/3874175/idea.log)

**Comment by Bombo** [comment]
This issue is not occurring with Android Studio 3.5.2, btw.

**Comment by ant-druha** [comment]
Sorry for the issue. It occurs if you run IDE under 1.8 JDK version. Please set IDE to run under 11 bundled JDK: https://intellij-support.jetbrains.com/hc/en-us/articles/206544879 as a workaround. Will be fixed in #38

**Comment by Bombo** [comment]
Thanks, that seems to be working.
